src/main/java/com/intershop/oms/test/servicehandler/omsdb/v1/OMSDbHandlerV1.java

```diff
@@ -3227,6 +3230,13 @@ private boolean doDBWaitForStateCheck(String debugType, String sqlStatementQuery
                 if (countRetry > 0)
                 {
                     Thread.sleep(retryDelay);
+                    countRetry = countRetry + 1;
```

Comment:
Warum zählst du hier hoch?

Reply:
Für die Fehlermeldung, falls es abbricht:
An Hikari pool connection isn't valid anymore while waiting for an object state, after " + countRetry + " attempts.

Reply:
Damit halbierst du aber die Anzahl/Zeit!